frontend/src/components/Navigator/components/SideNav.js

```diff
@@ -110,14 +107,13 @@ function SideNav({
             Autosaved on:
             <br />
             <ul className="margin-y-0">
-              {(lastSaveTime && connectionActive)
-                ? (
-                  <li>
-                    our network at
-                    {' '}
-                    {lastSaveTime.format(DATE_DISPLAY_SAVED_FORMAT)}
-                  </li>
-                ) : null}
+              {lastSaveTime && (
```

Comment:
I think we want to show the last saved time if we have it, regardless of whether we have an active connection (which is very fragile and could be a liar)